src/renderer/components/App/index.tsx

```diff
@@ -152,6 +158,13 @@ function App() {
                             key: 'a32nx-fbw',
                             url: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw/A32NX-master-cfbw.zip',
                             isExperimental: true,
+                            warningContent:
+                                <>
+                                    <p>The custom fly-by-wire version contains an experimental handling system that more closely matches real-life controls in an A320neo. This system is in development and bugs are to be expected.</p>
+                                    <p>To understand what you are getting into and the potential issues you might experience, please read <a href="https://github.com/flybywiresim/a32nx/blob/fbw/docs/README.md">this guide</a>.</p>
```

Comment:
instead of `<a href=''>...</a>` use `<div onClick={() => shell.openExternal("https://github.com/flybywiresim/a32nx/blob/fbw/docs/README.md")}>...</div>`

Otherwise it will open inside Electron and you cannot go back at all.